packages/styles/src/components/form-check.scss

```diff
@@ -82,7 +83,7 @@
       border-radius: 50%;
 
       &:checked::after {
-        margin: spacing.$size-micro;
+        border: spacing.$size-micro solid color.$white;
```

Comment:
This is a fix part of this PR: https://github.com/swisspost/design-system/pull/2065